src/js/util.js

```diff
@@ -50,7 +48,7 @@ var Util;
          * See #591
          */
         isMetaCtrlKey: function (event) {
-            if ((this.isMac && event.metaKey) || (!this.isMac && event.ctrlKey)) {
+            if ((Util.isMac && event.metaKey) || (!Util.isMac && event.ctrlKey)) {
```

Comment:
Since this is within a function call, the code won't execute until some other code actually invokes the function, so it's safe to refer to `Util` here.